build.sbt

```diff
@@ -119,12 +122,13 @@ lazy val `evaluator-client` = (project in file("eval-client"))
   .settings(
     name := "evaluator-client",
     libraryDependencies ++= Seq(
-      "org.http4s"    %% "http4s-blaze-client" % v('http4s),
-      "org.http4s"    %% "http4s-circe"        % v('http4s),
-      "io.circe"      %% "circe-core"          % v('circeversion),
-      "io.circe"      %% "circe-generic"       % v('circeversion),
-      "org.scalatest" %% "scalatest"           % v('scalatest) % Test
-    )
+      "org.http4s"    %% "http4s-blaze-client" % v("http4s"),
+      "org.http4s"    %% "http4s-circe"        % v("http4s"),
+      "io.circe"      %% "circe-core"          % v("circeversion"),
+      "io.circe"      %% "circe-generic"       % v("circeversion"),
+      "org.scalatest" %% "scalatest"           % v("scalatest") % Test
+    ),
+    crossScalaVersions := Seq(scala_212, scala_213)
```

Comment:
We don't need to cross-compile this project, having Scala 2.13 would be enough

Reply:
Libraries with dependencies used in the `sbt-exercise` project need a version in Scala 2.12. That's why cross-compilation.